digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -629,13 +644,14 @@ Returns empty result as Json message, not as page not found
     val user = userProvider.getCurrentUser()
     val municipalities: Set[Int] = if (user.isOperator()) Set() else user.configuration.authorizedMunicipalities
     val typeId = params.getOrElse("typeId", halt(BadRequest("Missing mandatory 'typeId' parameter"))).toInt
+    val usedService = if(typeId == maintenanceRoadService.maintenanceRoadAssetTypeId) maintenanceRoadService else linearAssetService

```

Comment:
We could have a method to get the correct linear service depending in the type we have. Instead of have this if else in all the places. And it will be useful in the future when we add more custom linear assets.